proto/imuachain/appchain/subscriber/v1/tx.proto

```
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package imuachain.appchain.subscriber.v1;
```

Buf (1.47.2)

Files with package "imuachain.appchain.subscriber.v1" must be within a directory "imuachain/appchain/subscriber/v1" relative to root but were in directory "proto/imuachain/appchain/subscriber/v1".

(PACKAGE_DIRECTORY_MATCH)